### PR TITLE
1314 - Fixed double click on open accordion

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

When you double click an open accordion-header the it will close on the first click, but will not reopen on the second. the arrow will change to open on the second click. On the third click the arrow will flip back with the content remaining closed.

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/1314

**Steps necessary to review your pull request (required)**:

- Checkout this branch
- Run http://localhost:4000/components/accordion/example-index.html
- Click on accordion header
- Once open double click accordion header
- Should expect to not flicker and open even doing double click

### CODE REVIEW

File: src/components/accordion/accordion.js [hunk 1]
```diff
@@ -775,12 +777,16 @@ Accordion.prototype = {
 
       if (pane.hasClass('no-transition')) {
         for (let i = 0; i < pane.length; i++) {
-          pane[i].style.display = 'block';
-          pane[i].style.height = 'auto';
+          setTimeout(() => {
```
Comment (EdwardCoyle) [inline src/components/accordion/accordion.js]: This has an adverse effect of adding a delay to the expansion of any headers that are clicked a single time.  It also seems like the expand/collapse calls still stack up on each other:

dec 21 2018 11_06 am

Comment (ericangeles) [reply src/components/accordion/accordion.js]: @EdwardCoyle thanks for the suggestions! Anyway, I don't see that error occurring on my end though. I tried to stopPropagation when doing a double click in the event but it seems the error is still there.

**Comment by tmcconechy** [comment]
Oh also can merge master now to fix tests.

**Comment by ericangeles** [comment]
Thanks for all the inputs @EdwardCoyle & @tmcconechy. Will do the fix now.

**Pull request merged**